kaggle_environments/static/player.html

```diff
@@ -142,16 +140,18 @@
                 : 1;
             try {
               if (debug) console.time("render");
-              await renderer({
-                ...context,
-                frame,
-                height: ref.current.clientHeight,
-                hooks: preactHooks,
-                parent: ref.current,
-                preact,
-                styled,
-                width: ref.current.clientWidth,
-              });
+              if (typeof renderer === "function") {
```

Comment:
This is the only real change in this file and was needed to not need to revert: https://github.com/Kaggle/kaggle-environments/commit/b044d1cc687e53c0f42f6cee37a3e4ee1e113fc6

Basic issue is even though the renderer is set in the context so should be initialized, when this is loaded from an iframe it can become undefined due to the way we are passing in props